app/connectors/httpParsers/GetEmploymentExpensesHttpParser.scala

```diff
@@ -38,13 +39,16 @@ object GetEmploymentExpensesHttpParser extends DESParser {
           },
           parsedModel => Right(parsedModel)
         )
+        case NOT_FOUND =>
```

Comment:
jsonErrors above is not used and could be supplemented with _ to avoid warning.
Also there is no test for the method at all. May be we should fix that.